tests/cpp/http_tests.cpp

```diff
@@ -14,7 +14,6 @@
 void testImport(std::shared_ptr<exa::Socket> socket) {
     exa::Chunk chunk{};
 
-    std::cerr << "Creating http chunk reader" << std::endl;
```

Comment:
is there a reason why you need to remove this

Reply:
There is a CRAN section, which prohibits writing to anything else than R console.